pkg/controllers/disruption/helpers.go

```diff
@@ -145,15 +145,33 @@ func GetPodEvictionCost(ctx context.Context, p *v1.Pod) float64 {
 	return clamp(-10.0, cost, 10.0)
 }
 
-func filterByPrice(options []*cloudprovider.InstanceType, reqs scheduling.Requirements, price float64) []*cloudprovider.InstanceType {
+// MinimumRequirementsInfo provides info on incompatible minValue requirement key if exists and minimum number of
+// instanceTypes required to meet all the minValue requirements.
+type MinimumRequirementsInfo struct {
```

Comment:
Is there a way to surface this data that doesn't require the creation of an entirely new struct?

Reply:
Removed this.